cypress.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I guess its better to create a folder `test/e2e` and put cypress staff there. Do not pollute root folder of pwa common
@devbucket : what do you think?